package.json

```diff
@@ -97,7 +97,8 @@
     "uint8array-extras": "^0.5.0",
     "utm": "^1.1.1",
     "validate-color": "^2.2.4",
-    "zustand": "^4.4.6"
+    "zustand": "^4.4.6",
+    "expo-av": "~13.10.6"
```

Comment:
alright ;)

Reply:
@EvanHahn installing expo-related modules is usually done via `expo install ...`, which pins to patch versions because expo compat is very sensitive based on the SDK version